zeebe/backup-stores/gcs/src/main/java/io/camunda/zeebe/backup/gcs/GcsBackupStore.java

```diff
@@ -167,11 +170,13 @@ public static Storage buildClient(final GcsBackupConfig config) {
   }
 
   public static void validateConfig(final GcsBackupConfig config) {
+    Objects.requireNonNull(config);
     try (final var storage = buildClient(config)) {
       storage.list(config.bucketName(), BlobListOption.pageSize(1));
     } catch (final Exception e) {
-      throw new CouldNotAccessBucketException(
-          ERROR_MSG_ACCESS_FAILED.formatted(config.bucketName()), e);
+      LOG.warn(
+          "Unable to verify that the config is valid, initialization will continue as it can be a transient network issue",
+          e);
```

Comment:
:x: This validation ignores all failures. I think it is better to follow the validation approach used in S3BackupStore. 

There is no need to query GCS during validation. This way it does not fail when GCS is temporarily unavailable. But we have to do other validations, like if bucket name is configured etc. It would be good to create a throw away client to verify the configuration if it does not fail when GCS is not reachable.

Reply:
In class `GcsBackupConfig` those fields are validate in the constructor, hence that's why no validation was added in this function.
Considering that the exception in the issue's stacktrace was raised from the call to `list` I don't know if building the client can raise an Exception.
I can separate these two calls into two different try/catch blocks and raise an exception only if building the client fails.

I would keep the log line for the list call so if the bucket is not existing for example it's logged somewhere with the name configured.